app/views/groups/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="login-page">
+  <p style="color: green"><%= notice %></p>
+
+<div class="login-top">
+    <h2 class="login-title">TRANSACTIONS</h2>
+    <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "login-links", id: "signout" %>
+  </div>
+
```

Comment:
- [ ] Kindly follow the design and add a back icon that should allow the user to go back to the previous page

![image](https://user-images.githubusercontent.com/65068771/164567676-6368900e-a057-4c4f-914a-6041319545b6.png)